tutorials/intermediate/arranging_axes.py

```diff
@@ -0,0 +1,368 @@
+"""
+===================================
+Arranging multiple Axes in a Figure
+===================================
+
+Often more than one "Axes" is wanted on a figure at a time, usually
+organized into a regular grid.  Matplotlib has a variety of tools for
+working with grids of Axes that have evolved over the history of the library.
+Here we will discuss the tools we think users should use most often, the tools
+that underpin how Axes are organized, and mention some of the older tools.
+
+.. note::
+
+    We use "Axes" to refer to the object that contains plotting
+    artists and the x- and y-Axis.  See `<figure_parts>`_ for more details of
+    Matplotlib terminology.
```

Comment:
> Technically, a *Subplot* is an Axes located in a grid defined by `GridSpec`, but for all practical purposes you can regard *Axes* and *Subplot*  as synonym.

As one way of getting the Subplot topic out of the way.

Reply:
I did 
``` 
    Matplotlib uses *Axes* to refer to an *Artist* that contains plotting
    *Artists* and the x- and y-*Axis*.  Another term that is often used is
    "subplot", which refers to an *Axes* that is arranged in a grid.
    See `<figure_parts>`_ for more details of Matplotlib terminology.
```